ironfish/src/memPool/feeEstimator.ts

```diff
@@ -152,8 +198,17 @@ export class FeeEstimator {
     }
 
     fees.sort((a, b) => (a < b ? -1 : a > b ? 1 : 0))
+    const averageBlockSize =
+      this.queues[BLOCK_SIZE].reduce((a, b) => a + b.blockSize, 0) /
+      this.queues[BLOCK_SIZE].length
+    const blockSizeRatio = BigInt(
```

Comment:
Discussed elsewhere -- we should not use block size to _increase_ the fee rate: doing so would allow miners to artificially fill blocks with transactions with fees of 0 or 1 $ORE in order to increase fee rate estimates.